### ISSUE TITLE
 Cosmic files hangs when ctrl+v name when creating new file

### ISSUE THREAD

**Issue by VandaLHJ** [pop-os/cosmic-files#138]
Every single time i try to create new file in /home/.config/cosmic (then for example com.system76/CosmicAppletTime/v1) or when i try to ctrl+c during renaming, it hangs (process have to be hard killed and occupy 8,3% cpu).

Copy and Paste is rather good to have when working in GUI file app.

TLDR:
1. Open app.
2. Create file or rename file
3. Press ctrl+c or ctrl+v
4. It hangs.

**Comment by joshuamegnauth54** [comment]
I'm able to replicate this. `Files` also hangs when copying or pasting from the breadcrumbs editor.

Log:
```
[2024-05-05T06:42:25Z TRACE calloop::sources] [calloop] Processing events for source type calloop_wayland_source::WaylandSource<smithay_clipboard::state::State<dnd::DndSurface>>
[2024-05-05T06:42:25Z TRACE calloop::loop_logic] [calloop] Dispatching events for source #1
[2024-05-05T06:42:25Z TRACE calloop::sources] [calloop] Processing events for source type calloop_wayland_source::WaylandSource<smithay_clipboard::state::State<dnd::DndSurface>>
[2024-05-05T06:42:25Z TRACE calloop::loop_logic] [calloop] Dispatching events for source #1
[2024-05-05T06:42:25Z TRACE calloop::sources] [calloop] Processing events for source type calloop_wayland_source::WaylandSource<smithay_clipboard::state::State<dnd::DndSurface>>
[2024-05-05T06:42:25Z TRACE calloop::loop_logic] [calloop] Dispatching events for source #1
[2024-05-05T06:42:25Z TRACE calloop::sources] [calloop] Processing events for source type calloop_wayland_source::WaylandSource<smithay_clipboard::state::State<dnd::DndSurface>>
[2024-05-05T06:42:25Z TRACE calloop::loop_logic] [calloop] Dispatching events for source #1
[2024-05-05T06:42:25Z TRACE calloop::sources] [calloop] Processing events for source type calloop_wayland_source::WaylandSource<smithay_clipboard::state::State<dnd::DndSurface>>
[2024-05-05T06:42:25Z TRACE calloop::loop_logic] [calloop] Dispatching events for source #1
[2024-05-05T06:42:25Z TRACE calloop::sources] [calloop] Processing events for source type calloop_wayland_source::WaylandSource<smithay_clipboard::state::State<dnd::DndSurface>>
[2024-05-05T06:42:25Z TRACE calloop::loop_logic] [calloop] Dispatching events for source #1
[2024-05-05T06:42:25Z TRACE calloop::sources] [calloop] Processing events for source type calloop_wayland_source::WaylandSource<smithay_clipboard::state::State<dnd::DndSurface>>
[2024-05-05T06:42:25Z TRACE calloop::loop_logic] [calloop] Dispatching events for source #1
[2024-05-05T06:42:25Z TRACE calloop::sources] [calloop] Processing events for source type calloop_wayland_source::WaylandSource<smithay_clipboard::state::State<dnd::DndSurface>>
```

I couldn't find anything wrong after poking around the code, and the log seems to suggest that the problem may lay elsewhere.

**Comment by onlyreportingissues** [comment]
Same problem for cutting out (ctrl + x).

**Comment by ThsUsr** [comment]
I also encountered this problem when copying and pasting text into any input widget.

**Comment by WatchMkr** [comment]
Reported on libcosmic here https://github.com/pop-os/libcosmic/issues/417

**Comment by jackpot51** [comment]
Fixed in #214
